tests/tracer/test_trace_utils.py

```diff
@@ -382,10 +385,15 @@ def test_set_http_meta(
         assert http.METHOD not in span.get_tags()
 
     if url is not None:
+        if url.startswith("http://user"):
+            expected_url = url[: url.index(":")] + "://" + url[url.index("@") + 1 :]
```

Comment:
I think a comment here might be helpful.

Reply:
```suggestion
            # Remove any userinfo that may be in the original url
            expected_url = url[: url.index(":")] + "://" + url[url.index("@") + 1 :]
```

Reply:
This feels a little too engineered (i.e. we test our code against exactly what our code does). It would probably be better if we make the expected url part of the parameters instead?